client/src/app/components/stock-stats/stock-stats.component.html

```diff
@@ -0,0 +1,57 @@
+<mat-card>
+  <div class="stats-header">
+    <h2 class="title">Stats</h2>
+    <mat-icon>trending_up</mat-icon>
+  </div>
+
+  <div class="stats-container">
+    <div class="col-stat">
+      <div class="row-stat">
+        <div class="tooltip">
+          <span class="tooltiptext"> Open definition</span>
+          <span class="stat-label">Open </span>
+        </div>
+        <span id="statOpen">{{ open }}</span>
+      </div>
+      <mat-divider [inset]="true"></mat-divider>
+      <div class="row-stat">
+        <span class="stat-label">High </span>
+        <span id="statHigh">{{ high }}</span>
+      </div>
+      <mat-divider [inset]="true"></mat-divider>
+      <div class="row-stat">
+        <span class="stat-label">Low </span>
+        <span id="statLow">{{ low }}</span>
+      </div>
+      <mat-divider [inset]="true"></mat-divider>
+      <div class="row-stat">
+        <span class="stat-label">Vol. </span>
+        <span id="statVol">{{ volume }}</span>
+      </div>
+      <mat-divider [inset]="true"></mat-divider>
+    </div>
+    &nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
[Question]: Can you explain why you added 4 non breaking white spaces here?

Reply:
it is to have some space between the left column and right column of the stats. I could change it margin but this will require that I add a css class (one for the left column and one for the right column)

Reply:
Add a css class! This is a hack and is prone to weird errors in the future